docs/standard/microservices-architecture/index.md

```diff
@@ -6,37 +6,79 @@ ms.author: wiwagn
 ms.date: 06/06/2018
```

Comment:
Because the entire book is being updated, you should update the date here.

Reply:
Done!